src/webgl/light.js

```diff
@@ -169,6 +169,8 @@ p5.prototype.specularColor = function(v1, v2, v3) {
 
 /**
  * Creates a directional light with a color and a direction
+ *
+ * Maximum limit of directional possible: 5
```

Comment:
I think that it would be better to use a complete sentence in order to keep with the writing style of the rest of the reference.

Something to the effect of 'A maximum of 5 directional lights can be active at one time.`

Reply:
Done